### PR TITLE
test populate game method

### PR DESCRIPTION
**Pull request by rachaelsalter**
When running a test I get this error:
RuntimeError: Circular dependency detected while autoloading constant Game

### CODE REVIEW

File: app/models/game.rb [hunk 1]
```diff
@@ -1,5 +1,54 @@
-class Game < ActiveRecord::Base
+gclass Game < ActiveRecord::Base
```
Comment (mscoutermarsh) [inline app/models/game.rb]: there is a typo here. The extra `g` :smile:

**Comment by rachaelsalter** [comment]
Aw man lol. Thanks. I am now getting the same error Joe got a while ago where there is a duplicate email (caused by a fixture) and I'm trying to figure it out:
 1) Error:
GameTest#test_board_is_populated_with_32_peices_when_started:
ActiveRecord::RecordNotUnique: PG::UniqueViolation: ERROR:  duplicate key value violates unique constraint "index_players_on_email"
DETAIL:  Key (email)=() already exists.
: INSERT INTO "players" ("created_at", "updated_at", "id") VALUES ('2015-09-29 22:40:05', '2015-09-29 22:40:05', 298486374)

**Comment by rachaelsalter** [comment]
I disabled fixtures and that error went away but I think I am stuck until the STI branch is merged into master.  

1) Error:
GameTest#test_board_is_populated_with_32_peices_when_started:
NameError: uninitialized constant Game::Pawn
    app/models/game.rb:11:in `block in populate_board!'
    app/models/game.rb:10:in`populate_board!'
    test/models/game_test.rb:5:in `block in class:GameTest'

**Comment by josephferrairo** [comment]
The STI branch is all set and merged. Is this branch all set to be merged as well now that STI has been merged?

**Comment by rachaelsalter** [comment]
Still working on fixing errors with the test for this branch.

**Comment by rachaelsalter** [comment]
The codeship build passed and my test to populate the game. However when I try to manually create a game in the rails console I get an error. See attachment for details.

screen shot 2015-10-06 at 11 59 57 am
irb(main):001:0> g = Game.create(:name => "Test Game", :white_player_id => 1, :black_player_id => 1)
PG::UndefinedTable: ERROR:  relation "games" does not exist

**Pull request merged**